brambling/models.py

```diff
@@ -775,17 +784,44 @@ class Payment(models.Model):
     card = models.ForeignKey('CreditCard', blank=True, null=True)
     is_confirmed = models.BooleanField(default=False)
 
+    def refund(self, amount, issuer, dwolla_pin=None):
```

Comment:
What happens if someone refunds an order paid by check? I'm not sure I totally understand the code.